src/LinuxTracing/LibunwindstackUnwinderTest.cpp

```diff
@@ -73,7 +73,7 @@ constexpr pid_t kProcessId = 123;
 //     12be:       48 83 c4 10 c3              add    $0x10,%rsp
 
 TEST(LibunwindstackUnwinder, DetectsFunctionWithFramePointerSet) {
-  auto unwinder = LibunwindstackUnwinder::Create();
+  auto unwinder = LibunwindstackUnwinder::Create({});
```

Comment:
If the parameter is not given a default value, I prefer an explicit `nullptr` to `{}`. Passing `{}` instead of `nullptr` is a bit like passing `{}` instead of `0` when an `int` is expected: correct but weird to me.

Reply:
That was also a leftover from when I was passing the map in as value. Anyways. With the default parameter, it is now obsolete anyways.